### PR TITLE
Unit 3 - Functions - Eddie Tong

### PR DESCRIPTION
**Pull request by Edison-Tong**
functions

### CODE REVIEW

File: Unit 3/functions/quiz.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Functions Quiz
+
+1. Explain what D.R.Y. code is.
+   Stand for DO NOT REPEAT meaning: make your code consise
```
Comment (AcquirEdge) [inline Unit 3/functions/quiz.md:4]: Actually stands for Don't Repeat Yourself. Close, but please fix this.

**Review by AcquirEdge — CHANGES_REQUESTED** [review]
Just one change to your quiz.

**Comment by Edison-Tong** [comment]
HA I don't know what I was thinking with that answer. I got it fixed now

**Review by AcquirEdge — APPROVED** [review]
Looks good!

**Pull request merged**